src/main/java/net/sf/jabref/exporter/BibDatabaseWriter.java

```diff
@@ -289,7 +289,7 @@ private void writeMetaData(Writer out, MetaData metaData) throws IOException {
         // write groups if present. skip this if only the root node exists
         // (which is always the AllEntriesGroup).
         GroupTreeNode groupsRoot = metaData.getGroups();
-        if ((groupsRoot != null) && (groupsRoot.getChildCount() > 0)) {
+        if ((groupsRoot != null) && (groupsRoot.getNumberOfChildren() > 0)) {
```

Comment:
There is already `isLeaf()` which indicates whether a node as children or not. Not sure if `root.isLeaf()` is easier to understand than `root.numberOfChildren > 0`. Will keep your comment in mind when I`ll revisit the code.